README.md

```diff
@@ -122,12 +122,16 @@ publicClient.getTime(callback);
 The [private exchange API
 endpoints](https://docs.gdax.com/#private) require you to
 authenticate with an API key. You can create a new API key [in your exchange
-account's settings](https://gdax.com/settings).
+account's settings](https://gdax.com/settings). You can also specify the 
+environment.
 
 ```javascript
 var Gdax = require('gdax');
+var apiURI = 'https://api.gdax.com';
```

Comment:
Perhaps a comment here to indicate that this parameter is optional: 
// Defaults to `https://api.gdax.com` if omitted